de-lib/src/main/java/org/iplantc/de/diskResource/client/views/metadata/DiskResourceMetadataViewImpl.java

```diff
@@ -295,6 +297,11 @@ void onImportSelected(SelectEvent event) {
         presenter.onImport(additionalMdgrid.getSelectionModel().getSelectedItems());
     }
 
+    @UiHandler("saveToFileButton")
+    void onSaveToFileButtonSelected(SelectEvent event) {
+         presenter.onSaveToFile();
```

Comment:
It'd be nice if the view fired off its own event instead of calling the presenter, but since this whole class is calling the presenter, that might be a refactor saved for another time.